ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -173,9 +173,16 @@
             <date-part name="year"/>
           </date>
           <group>
-            <text term="volume" form="short" suffix=".&#160;"/>
-            <text variable="volume"/>
-            <text variable="number-of-volumes" prefix="/"/>
+            <choose>
+              <if variable="volume" match="none">
+                <text variable="number-of-volumes" suffix="&#160;vol."/>
```

Comment:
thanks . I will try that right away !

2015-12-10 6:07 GMT+01:00 Rintze M. Zelle <notifications@github.com>:

> In ens-de-lyon-centre-d-ingenierie-documentaire.csl
> <https://github.com/citation-style-language/styles/pull/1810#discussion_r47189463>
> :
>
> > @@ -173,9 +173,16 @@
> >              <date-part name="year"/>
> >            </date>
> >            <group>
> > -            <text term="volume" form="short" suffix=".&#160;"/>
> > -            <text variable="volume"/>
> > -            <text variable="number-of-volumes" prefix="/"/>
> > +            <choose>
> > +              <if variable="volume" match="none">
> > +                <text variable="number-of-volumes" suffix="&#160;vol."/>
>
> @smarchand <https://github.com/smarchand>, in general it's better to use
> CSL terms (and groups) whenever possible, by the way. Here you could have
> used
>
> <group delimiter="&#160;">
>   <text variable="number-of-volumes"/>
>   <text term="volume" form="short"/>
> </group>
>
> instead of
>
> <text variable="number-of-volumes" suffix="&#160;vol."/>
>
> We can leave this as is, but something to keep in mind for future edits.
> (using terms make it possible to automatically translate styles into other
> languages)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/styles/pull/1810/files#r47189463>
> .
>



-- 
Stéphane